Governance/tech/permissions.md

```diff
@@ -382,11 +387,11 @@ this will preserve the ACLs. If run with the additional archive flag `-raA`, the
 
 ```{note}
 The -A flag requires source and destination to have compatible ACL systems, when this is not the case a potential solution in rsync is to use the `--chmod` option.
-From the rsync man page (in the --perms section):
+From the rsync main page (in the --perms section):
```

Comment:
Nope, reject this change, `man` is correct.

Reply:
Given that I'm confident this doesn't refer to gender and I can't guess what else it could be, maybe a short explanation needed?

Reply:
man stands for "manual" is a command to show the help on unix commands, ie. in this case "man rsync" will show all the options.  However, if you're a unix user you would know about man

Reply:
Could probably add the word "manual" in there somewhere, e.g. "(in the --perms section of the manual)"